python/dataproc_templates/gcs/README.md

```diff
@@ -63,3 +63,76 @@ export JARS="gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
     --gcs.bigquery.output.mode=<append|overwrite|ignore|errorifexists> \
     --gcs.bigquery.temp.bucket.name="<temp-bq-bucket-name>"
 ```
+
+## GCS To Spanner
+
+Template for reading files from Google Cloud Storage and writing them to a
+Spanner table. It supports reading Parquet, Avro and ORC formats.
+
+## Arguments
+
+* `gcs.spanner.input.location`: GCS location of the input file
+(format: `gs://BUCKET/...`)
+* `gcs.spanner.input.format`: Input file format (one of: avro, parquet, orc)
+* `gcs.spanner.output.instance`: Spanner instance name
+* `gcs.spanner.output.database`: Spanner database name
+* `gcs.spanner.output.table`: Spanner table name
+* `gcs.spanner.output.primary_key`: Primary Key of the Spanner table 
```

Comment:
We should add the fact that the PK is only required for new tables here

Reply:
Agree, and in the Java implementation as well: https://github.com/GoogleCloudPlatform/dataproc-templates/blob/main/java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCSToSpannerConfig.java#L75

Reply:
Good catch! Let's make the Python template as user-friendly as possible and then backport the improvements to java on a different PR.

Reply:
Done in 947947d